kube/helm/app/templates/ingress.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Chart.Name }}
+  namespace: vegait-training
+  annotations:
+    kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/target-type: ip
+    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:eu-central-1:162340708442:certificate/7295afe7-3e51-4271-89eb-4d7bd8192571
```

Comment:
You don't need this, LB will find cert provisioned via ACM automatically based on the host that you specified here in ingress.